blog-posts/emails/article.md

```diff
@@ -0,0 +1,329 @@
+          </mj-text>
+          <mj-text font-family="Trebuchet MS" color="#173940" font-size="16px" align="center">
+            Click the button bellow to log in:
+          </mj-text>
+          <mj-button background-color="#F48668" border-radius="20px" font-size="16px" font-weight="600" href="http://www.pchol.fr">Join my app</mj-button>
+        </mj-column>
+      </mj-section>
+    </mj-wrapper>
+    <mj-wrapper border-radius="8px" padding="15px">
+      <mj-section background-color="#FFFFFF" border-radius="8px">
+        <mj-column>
+          <mj-text font-family="Trebuchet MS" color="#173940" font-size="16px" align="center">
+            Ran into a problem ? Do not hesitate to contact me at <a href="mailto:help@pchol.fr"><b>help@pchol.fr</b></a>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+    </mj-wrapper>
+  </mj-body>
+</mjml>
+```
+
+- The `<mj-head>` part allows me to:
+  - Specify my preferred color scheme (only light). [Advanced MJML tricks][advanced-mjml] include light and dark display compatibility. Check it out!
+  - Write some css to further style my contact email address. I try to keep it minimal, as it won't be covered by MJML compatibility features and may not work on some clients.
+- The `<mj-body>` part describes what will be displayed. It's not html but it remains fairly easy to understand. Check [the documentation][mjml-documentation] to learn more about this syntax!
+
+By compiling your MJML template into html + css, you can use it in a SES template (or later in raw emails). To compile, either use the hands-on [live editor][mjml-live-editor], or go with npm:
+
+```sh
+npm run mjml myTemplate.mjml --output myTemplate.html
+```
+
+Let's have an look on the result! No more big differences between email clients. Except for the contact email address (I warned you 🤓), the result is just what was expected.
+
+![MacOS Mail vs Gmail MJML](./assets/mjml-mac-gmail-difference.png 'MacOS Mail vs Gmail MJML')
+
+### The power of raw emails: attachments, images, custom headers and more
+
+Everything that was covered until now can be implemented using either templated or raw SES emails. But from now, you will be diving into the unique possibilities offered by raw emails, to push your messages to their limit.
+
+Let's say I want to add a logo in my email's header. Using templated emails, it will be impossible to have it displayed on every mail client. The solution is to send a raw email with your image as attachment, and to reference it in the message's body, using contentID.
+
+I modified my email's header MJML code to include an image:
+
+```xml
+<mj-section background-color="#F48668" border-radius="8px 8px 0 0" padding="0">
+  <mj-column>
+    <mj-image width="20px" src="cid:my-logo@pchol.fr" alt="My logo"/>
+  </mj-column>
+</mj-section>
+```
+
+Then, in the lambda I use to send my emails, I can use the Nodemailer library to translate my MJML template into a raw email, and to add attachments to my message.
```

Comment:
Could be interesting to see the raw email structure, with multiple section (plain text and html separated by contentId), and on this occasion, succintly introduce MIME messages which is also a key concept when talking about emails

Reply:
And also say that some client will display html, but if they can't (name one), they will use plain text and thus your message will display anyway !